server/neptune/workflows/internal/terraform/response.go

```diff
@@ -0,0 +1,7 @@
+package terraform
+
+import "github.com/runatlantis/atlantis/server/core/config/valid"
+
+type Response struct {
+	FailedPolicies map[string]valid.PolicySet
```

Comment:
I think we can model this a bit better. Current approach assumes we might only ever care about failing policies for the forseeable future.  I'd rather return a list of objects containing the result and the policy set;
e.g.
```go
type ValidationStatus int
const (
    Success ValidationStatus = iota
    Fail ValidationStatus
    Warn ValidationStatus
)
type ValidationResult struct {
    Status ValidationStatus
    PolicySet valid.PolicySet
}
```

Even if you're only passing in failed policies right now, it's future proof.

Also sidenote: i don't necessarily want us using objects from `valid` pkg because changing those can have unintended consequences in our workflow.  I'd rather keep objects here separate. if possible, can we change this?

Reply:
Replaced FailedPolicies with ValidationResults, though I skipped the Warn status for now (it would require the Conftest activity being aware that the [warn flag](https://www.conftest.dev/options/#-fail-on-warn) + processing error exit codes). 

Unsure I understand what you mean by the consequences of using the `valid` pkg here. Wouldn't we want to rely on `valid` to be our source of truth for policy definitions and change our activity should the `PolicySets` field change in the future?

Reply:
> Unsure I understand what you mean by the consequences of using the valid pkg here. Wouldn't we want to rely on valid to be our source of truth for policy definitions and change our activity should the PolicySets field change in the future?

It is the source of truth, but we specifically build out the workflow request with objects that are isolated to the workflow so that we're in complete control of what we're changing.  You're using an object directly that represents server side config, if you change that object without thinking about how it's going to affect our workflow, there could be issues. Let's just build another policy set object that's isolated to the workflow.

Reply:
Also went ahead and decided to mark the check run status as success vs. failure in the event we process failed policies.